handlerBasic.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+func handlerGetBasic(w http.ResponseWriter, r *http.Request) {
+	// Extracting the latitude and longitude from the query string
+	log.Println(r.URL.Query())
+	queryStr := r.URL.Query()
+	if queryStr["lat"] == nil || queryStr["lon"] == nil {
+		// Respond with error in JSON
+		log.Fatal("Missing Coordinates")
+	}
+	log.Println(queryStr)
+	client := http.Client{
+		Timeout: time.Duration(2) * time.Second,
+	}
+	resp, err := client.Get(fmt.Sprintf("https://my.meteoblue.com/packages/basic-15min_basic-day_wind-15min_wind-day?apikey=dJK6DxymAIS4Oqcz&lat=%v&lon=%v&format=json", queryStr["lat"][0], queryStr["lon"][0]))
+	defer resp.Body.Close()
+	if err != nil {
+		// Respond with Error
+		log.Fatal("Oh NORR Error, could't get response", err)
```

Comment:
In the vast majority of cases `log.Fatal` should only be used in main, or init functions. Since an API request failing doesn't hurt this web servers dependency to continue serving other requests, it must not exit on error; it calls `os.Exit` and the program dies.

Reply:
Ahhh ok, noted.